addons/udes_stock/models/stock_picking_batch.py

```diff
@@ -506,16 +503,19 @@ def get_user_batches(self):
         return batches
 
     def unassign_user_batches(self):
-        """ Removes pickings from all batches assigned to the user and cancels the batch
+        """ Get batches for user and unassign them
```

Comment:
You're also removing the incomplete pickings.

Reply:
yes! unassigning a batch is to do just that - if it's ephemeral